dash.el

```diff
@@ -1591,6 +1591,54 @@ and when that result is non-nil, through the next form, etc."
                    (--> ,result ,form))
                  ,@more))))
 
+(defmacro -cond-> (x &rest clauses)
+  "Conditionally thread X through CLAUSES.
+Threads x (via `->') through each form for which the
+corresponding test expression is true.  Note that, unlike cond
+branching, `-cond->' threading does not short circuit after the
+first true test expression."
```

Comment:
Some nits/suggestions:
```
x -> X
true -> non-nil
cond -> `cond'
branching -> clauses
```
Maybe it's just my unfamiliarity with Clojure, but it's not clear to me what "test expression" is referring to. I would appreciate an explanation of how it relates to `CLAUSES`, which is mentioned in the beginning but not subsequently explained. I think some destructuring like in the docstring of `cond` would be helpful, as would mentioning the expected length of `CLAUSES`.